src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -56,6 +57,9 @@ async def send_run_change_to_mode_cmd(self, newMode) -> Clusters.Objects.RvcRunM
     def write_to_app_pipe(self, command):
         with open(self.app_pipe, "w") as app_pipe:
             app_pipe.write(command + "\n")
+        # Allow some time for the command to take effect.
+        # This removes the test flakyness which is very annoying for everyone in CI.
```

Comment:
add TODO with #32139 here and below saying that sleep should be gone.

We should replace sleep with some way of polling of "Did it take effect". why is 0.001 correct? why not 0.00001 ? why not 1?

Reply:
Added a TODO.